src/main/io/ledstrip.c

```diff
@@ -1006,11 +1022,13 @@ void ledStripInit(ledConfig_t *ledConfigsToUse, hsvColor_t *colorsToUse, failsaf
     ledStripInitialised = false;
 }
 
+#define ROTATION_SEQUENCE_LED_WIDTH 0x03       // 0x03 = 2 leds ON ; 0x05 = 3 leds ON
+
 void ledStripEnable(void)
 {
     reevalulateLedConfig();
     ledStripInitialised = true;
-
+    maskRotationPhase = ROTATION_SEQUENCE_LED_WIDTH | (ROTATION_SEQUENCE_LED_WIDTH << (ledsInRingCount>>1));
```

Comment:
Ok for that too. Only, I would prefer "ledsInRingCount >> 1" rather than "ledsInRingCount / 2". It optimises the compilator function.